File: tugas/php/Ram.php
<?php

class Ram{
    // atribut
    private $capacity, $price;

    // constructor
    public function __construct($capacity = 0, $price = 0){
        $this->capacity = $capacity;
        $this->price = $price;
    }

    // setter n getter
    public function setCapacity($capacity){
        $this->capacity = $capacity;
    }

    public function getCapacity(){
        return $this->capacity;
    }

    public function setPrice($price){
        $this->price = $price;
    }

    public function getPrice(){
        return $this->price;
    }
}

File: tugas/java/Ram.java
public class Ram {
    // atribut
    private int capacity;
    private int price;

    // constructor
    public Ram(){}

    public Ram(int capacity, int price) {
        this.capacity = capacity;
        this.price = price;
    }

    // setter n getter
    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }
    
    public int getCapacity() {
        return capacity;
    }
    
    public void setPrice(int price) {
        this.price = price;
    }

    public int getPrice() {
        return price;
    }
}